### DISCUSSION TITLE
Merge logging-operator & logging-operator-logging charts

### DISCUSSION THREAD

**Discussion by pepov** [kube-logging/logging-operator#1389] [General]
These charts are separate due to historical reasons. We should merge them together and then we can move it back to this repository and setup an OCI based release process for it.

**Comment by pepov** [comment]
I couldn't complete a fully compatible merge, but most of the functionality of the `logging-operator-logging` chart has been merged into the `logging-operator` chart recently:
https://github.com/kube-logging/logging-operator/pull/1382

We are going to add features and fix bugs in this chart only in the future and will deprecate the old charts and the old chart repository, since now we are going to publish our charts into ghcr.io as OCI artifacts, see https://github.com/kube-logging/logging-operator/issues/1352

(converted to a discussion to be able to discuss any issues coming up about the changes here first)

**Comment by eevdev** [comment]
Hi @pepov, if the [kube-logging/logging-operator-logging chart](https://github.com/kube-logging/helm-charts/tree/main/charts/logging-operator-logging) is soon to be deprecated it would be nice to have some migration instructions.

Some initial observations after comparing generated manifests:
- need to move most Helm values to `logging` section
- need to disable rbac (`rbac.enabled: false`)
- bug: no way to disable creation of Service and Deployment

Background: I need to install Logging, (Cluster)Flow and (Cluster)Output, and can not install those from the same Helm install as the logging-operator itself.

**Reply by pepov** [reply]
In the release notes we've added a few notes on this:
https://github.com/kube-logging/logging-operator/releases/tag/4.3.0

Specifically:
https://github.com/kube-logging/helm-charts/blob/main/charts/logging-operator-logging/README.md
